src/EnergyPlus/WindowManager.cc

```diff
@@ -1162,10 +1141,6 @@ namespace WindowManager {
                 Phi = double(IPhi - 1) * 10.0;
                 for (IGlass = 1; IGlass <= NGlass; ++IGlass) {
                     LayPtr = Construct(ConstrNum).LayerPoint(LayerNum(IGlass));
-                    if (Material(LayPtr).GlassSpecAngTransDataPtr) {
-                        Phi = TableLookup(PerfCurve(Material(LayPtr).GlassSpecAngTransDataPtr).TableIndex).X1Var(IPhi);
-                        break;
-                    }
```

Comment:
Same issue before. This section is not useful after the section is removed.

Reply:
This one is useless. I will remove it.